### PR TITLE
Looking for contributors

### PR DESCRIPTION
**Pull request by radumarias**

### CODE REVIEW

File: draft/2024-07-31-this-week-in-rust.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 [guidelines]:https://github.com/rust-lang/this-week-in-rust?tab=readme-ov-file#call-for-participation-guidelines
 
+Hi, I have a few OSS projects in need for contributors. More details [here](https://users.rust-lang.org/t/looking-for-contributors-to-open-source-projects/115191).
+
```
Comment (colelawrence) [inline draft/2024-07-31-this-week-in-rust.md:91]: Consider reading the guidelines to be included https://github.com/rust-lang/this-week-in-rust?tab=readme-ov-file#call-for-participation-guidelines

Comment (radumarias) [reply draft/2024-07-31-this-week-in-rust.md:91]: sorry bit I did, can you please tell me what I'm doing wrong

Comment (nellshamrell) [reply draft/2024-07-31-this-week-in-rust.md:91]: Hello! Ty for submitting this :) 

The format we prefer is links to individual issues on a project, [you can see a good example here ](https://github.com/rust-lang/this-week-in-rust/blob/621ded70e552893cb7ae126004c0ee4bb09f04c0/content/2024-01-03-this-week-in-rust.md?plain=1#L81)

Do you mind picking out a few issues for a few of those projects and submitting those to next week's issue?

Thank you!

Comment (radumarias) [reply draft/2024-07-31-this-week-in-rust.md:91]: > Hello! Ty for submitting this :)
> 
> The format we prefer is links to individual issues on a project, [you can see a good example here ](https://github.com/rust-lang/this-week-in-rust/blob/621ded70e552893cb7ae126004c0ee4bb09f04c0/content/2024-01-03-this-week-in-rust.md?plain=1#L81)
> 
> Do you mind picking out a few issues for a few of those projects and submitting those to next week's issue?
> 
> Thank you!

thank you for your response, smth like this?
https://github.com/radumarias/rencfs/issues?q=is%3Aissue+label%3Afirst-timers-only+label%3A%22good+first+issue%22+is%3Aopen

or create a list of links with all of them? it's ok to post here or where exactly?

Comment (nellshamrell) [reply draft/2024-07-31-this-week-in-rust.md:91]: I would pick out just a few specific issues at first (maybe 2-3 for up to 4 projects), and then we can include more in future issues. Could you open a new pull request against the current draft?

Comment (nellshamrell) [reply draft/2024-07-31-this-week-in-rust.md:91]: [This is the current draft](https://github.com/rust-lang/this-week-in-rust/blob/master/draft/2024-08-07-this-week-in-rust.md)

Comment (radumarias) [reply draft/2024-07-31-this-week-in-rust.md:91]: Thank you, I created this https://github.com/rust-lang/this-week-in-rust/pull/5790

**Comment by bennyvasquez** [comment]
closing this one in favor of #5790. thank you!

**Comment by radumarias** [comment]
I want to thank you for my PR getting to newsletter, I really appreciate it

**Pull request closed without merging**